### ISSUE TITLE
Why training ends at epoch 50?

### ISSUE THREAD

**Issue by yuxudong20** [microsoft/ATAC#4]
Hello, I  have tried to reproduce ATAC's results in the paper. However, when I run the official codes, the experiment automatically ends at epoch 50. I cannot find where the problem is? Could you give me some help?
For example, I have run 'python scripts/main.py -e hopper-medium-expert-v2 --gpu_id 0 --seed 15'. Are there any other hyperparameters that need to be given?
@chinganc

**Comment by chinganc** [comment]
Hi @yuxudong20, there was pytorch bug created in our refactoring which causes errors when using gpu. It's fixed now. Can you please give it a try? Thanks.

**Comment by yuxudong20** [comment]
And the amplification shows that it falls down at epoch 50.
![image](https://user-images.githubusercontent.com/47274016/190417816-3a20f6d4-2d10-4a50-98ce-000d5dcb3ab2.png)

**Comment by chinganc** [comment]
This is normal: Currently, it [by default](https://github.com/microsoft/ATAC/blob/af5071e923f93b28e684dd241f4b6c414ab3bd76/scripts/main.py#L221) uses 50 epochs to do warmstart, which is just BC for the policy, and then ATAC starts. The drop you see there is the first epoch of ATAC.
